src/Features/Core/Portable/Completion/CommonCompletionItem.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis.Completion
 {
     internal static class CommonCompletionItem
     {
-        [Obsolete("This is a compatibility shim for FSharp; please do not use it.", error: true)]
+        [Obsolete("This is a compatibility shim for FSharp; please do not use it.")]
```

Comment:
This can't be an error because we need to use it and we don't have the shim yet.

Reply:
❗️ @TIHan please revert this change. Mark the code where you need to use this as `[Obsolete]` and it will not produce a build error.

Reply:
Ok